packages/nc-gui-v2/pages/index/index/create-external.vue

```diff
@@ -77,51 +93,91 @@ const validators = computed(() => {
 
 const { validate, validateInfos } = useForm(formState, validators)
 
+const populateName = (v: string) => {
+  formState.dataSource.connection.database = `${v.trim()}_noco`
+}
+
 const onClientChange = () => {
   formState.dataSource = { ...getDefaultConnectionConfig(formState.dataSource.client) }
+  populateName(formState.title)
 }
 
-const inflectionTypes = ['camelize', 'none']
-const configEditDlg = ref(false)
+const onSSLModeChange = ((mode: 'No' | 'Allow' | string) => {
+  switch (mode) {
+    case 'No':
+      delete formState.dataSource.connection.ssl
+      break
+    case 'Allowed':
+      formState.dataSource.connection.ssl = 'true'
+      break
+    default:
+      formState.dataSource.connection.ssl = {
+        ca: '',
+        cert: '',
+        key: '',
+      }
+      break
+  }
+}) as unknown as SelectHandler
 
-// populate database name based on title
-watch(
-  () => formState.title,
-  (v) => (formState.dataSource.connection.database = `${v?.trim()}_noco`),
-)
+const updateSSLUse = () => {
+  if (formState?.dataSource?.connection?.ssl) {
+    if (typeof formState.dataSource.connection.ssl === 'string') {
+      formState.sslUse = 'Allowed'
```

Comment:
Are these required to be Capitalized words? If not, I would be glad if we change it to lowercase `no` `allowed` etc. because it's easier to avoid mistakes when everything is lowercase.

Even better yet, if we can instead use an Enum for this that'd make it optimal I'd say (then you can keep the Capitalized words, it wouldn't matter)

```ts
enum SSLUse {
	No,
	Allowed,
	Preferred,
}
```